docs/features/fiat-rates.md

```diff
@@ -4,23 +4,23 @@ Suite provides several types of fiat rates:
 
 -   Current fiat rate: Used on Dashboard, next to the account balance, for converting a crypto amount in send form to a fiat currency, etc...
 -   Weekly rates: In addition to current rate we also fetch 7 days old rate and based on the difference we either show green or red arrow next to an exchange rate (can be seen in assets table on Dashboard).
--   Historical fiat rate: Exchange rate at the time of facilitating a transaction. Used in list of transactions to calculate daily deltas and in transaction detail modal. Not supported for ERC20 tokens.
+-   Historical fiat rate: Exchange rate at the time of facilitating a transaction. Used in list of transactions to calculate daily deltas and in transaction detail modal.
 
 ## Providers
 
--   Blockbook: For all main networks (BTC, LTC, ETH) except XRP and ERC20 tokens
+-   Blockbook: For all main networks (BTC, LTC, ETH + ERC-20 tokens, ...) except XRP and ADA
 -   Blockbook "live" updates: Apart from using Blockbook to manually fetch fiat rates, it also provides us with "live" fiat rate updates (basically push notifications via websocket) for all blockbook-supported coins (see above).
```

Comment:
No longer true, right? This PR removes live subscription for fiat rates.

Reply:
It was updated to 

```
-  Blockbook: For all main networks (BTC, LTC, ETH + ERC-20 tokens, ...) except XRP, SOL + SOL tokens and ADA + ADA tokens
-   CoinGecko: Used for XRP, SOL + SOL tokens and ADA + ADA tokens and as a fallback for failed requests to blockbook.
```